s6/usr/local/sbin/start_api

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+while :
```

Comment:
more visibility with pre-echo like in `s6/usr/local/sbin/start_jobs` will be great AFM)